github/DependabotAlert.py

```diff
@@ -0,0 +1,173 @@
+                "fixed_at": self._fixed_at.value,
+                "html_url": self._html_url.value,
+                "number": self._number.value,
+                "security_vulnerability": self._security_vulnerability.value,
+                "state": self._state.value,
+                "updated_at": self._updated_at.value,
+                "url": self._url.value,
+            }
+        )
+
+    @property
+    def created_at(self):
+        """
+        :type: string
+        """
+        return self._created_at.value
+
+    @property
+    def dependency(self):
+        """
+        :type: :class:`github.Dependency.Dependency`
+        """
+        return self._dependency.value
+
+    @property
+    def dismissed_at(self):
+        """
+        :type: string
+        """
+        return self._dismissed_at.value
+
+    @property
+    def dismissed_by(self):
+        """
+        :type: :class:`github.NamedUser.NamedUser`
+        """
+        return self._dismissed_by.value
+
+    @property
+    def dismissed_comment(self):
+        """
+        :type: string
+        """
+        return self._dismissed_comment.value
+
+    @property
+    def dismissed_reason(self):
+        """
+        :type: string
+        """
+        return self._dismissed_reason.value
+
+    @property
+    def fixed_at(self):
+        """
+        :type: string
+        """
+        return self._fixed_at.value
+
+    @property
+    def html_url(self):
+        """
+        :type: string
+        """
+        return self._html_url.value
+
+    @property
+    def number(self):
+        """
+        :type: integer
+        """
+        return self._number.value
+
+    @property
+    def security_vulnerability(self):
+        """
+        :type: :class:`github.SecurityVulnerability.SecurityVulnerability`
+        """
+        return self._security_vulnerability.value
+
+    @property
+    def state(self):
+        """
+        :type: string
+        """
+        return self._state.value
+
+    @property
+    def updated_at(self):
+        """
+        :type: string
+        """
+        return self._updated_at.value
+
+    @property
+    def url(self):
+        """
+        :type: string
+        """
+        return self._url.value
+
+    def _initAttributes(self):
+        self._created_at = github.GithubObject.NotSet
+        self._dependency = github.GithubObject.NotSet
+        self._dismissed_at = github.GithubObject.NotSet
+        self._dismissed_comment = github.GithubObject.NotSet
+        self._dismissed_reason = github.GithubObject.NotSet
+        self._fixed_at = github.GithubObject.NotSet
+        self._html_url = github.GithubObject.NotSet
+        self._number = github.GithubObject.NotSet
+        self._security_vulnerability = github.GithubObject.NotSet
+        self._state = github.GithubObject.NotSet
+        self._updated_at = github.GithubObject.NotSet
+        self._url = github.GithubObject.NotSet
```

Comment:
You're missing:
- `security_advisory`
- `dismissed_by`
- `auto_dismissed_at`

Not sure if that was intentional

Reply:
Security advisory intentionnaly left out since it is a huge object and not needed for our use case.

Fixed missing dismissed_by attributes. It was missing indeed.

I added auto dismissed_at added although not used in our case.

Reply:
Sounds good. My aim here is to increase the likelyhood that this gets merged upstream eventually. But if you don't need them and they slow you down, we can go like this